### ISSUE TITLE
Memory usage giving negative values

### ISSUE THREAD

**Issue by lunik1** [enkore/i3pystatus#485]
Running the latest git master on Arch Linux, the mem module is currently displaying negative values which don't seem to have any bearing on the actual memory usage. He's the relevant part of my i3status.py:

```
status.register("mem",
                format="𐇭  {used_mem:.0f} MiB")
```

**Comment by hcpl** [comment]
@lunik1, looks like the calculation formula has changed. Here's the culprit:
```python
from psutil import virtual_memory
...
memory_usage = virtual_memory()
used = memory_usage.used - memory_usage.cached - memory_usage.buffers
...
cdict = {
    "used_mem": used / self.divisor,
    ...
}
```

**Comment by hcpl** [comment]
@lunik1, done: #486.

**Comment by lunik1** [comment]
Seems to have fixed it, cheers!

**Comment by enkore** [comment]
Fixed by #486
